lib/widgets/global/surface/surface_layer.dart

```diff
@@ -66,72 +68,13 @@ class BoxSurface extends StatelessWidget {
           // Create outline around the surface
           side: outline
               ? BorderSide(
-                  color: context.theme.colorScheme.background.op(0.2),
-                  width: 2,
+                  color: theme.foregroundColor.op(0.25),
```

Comment:
Probs wanna make this something around 0.1. Looks way better on a darker background.